internal/licenseutils/utils.go

```diff
@@ -159,12 +160,38 @@ func LoadLocalIssuedLicense(ctx context.Context, filename string) (*model.Issued
 	return doLoadLocalIssuedLicense(ctx, filename, lclient)
 }
 
+// GetLicenseSummary summarizes the license for the user
+func GetLicenseSummary(ctx context.Context, license model.IssuedLicense) (string, error) {
```

Comment:
I see a lot of code duplication between `GetLicenseSummary` and `LoadLocalIssuedLicense/doLoadLocalIssuedLicense `. I guess a function which creates a new `licensing.Config` and returns a verified license could be extracted here.